### DISCUSSION TITLE
Journal Imort & Local LLM

### DISCUSSION THREAD

**Discussion by maliikm** [UdaraJay/Pile#105] [Ideas]
I think Pile is on to something and one incredible feature that is missing right now is the ability to import pages from physical journals. I have three journals that I've kept since middle school (over ten years) that are filled with writings that I haven't revisited in years. 

It would be so much fun if I could use Pile and AI to explore these pages again. Also, it would be great to see Pile come with ability to install (or come pre-installed) with an open source local LLM to help keep things private, as I'm sure there are some things in my journals that I don't want OpenAI seeing. That's the only major downside of the app as-is, but for the most part, there are some incredible ideas here.

This is an awesome project Udara, I hope you can continue developing it one day.

**Comment by UdaraJay** [comment]
Import is a nice idea! 

Local llms are supported via ollama– check the ollama settings tab for instructions on how to use it with a local model.
